lib/_http_server.js

```diff
@@ -305,20 +306,16 @@ ServerResponse.prototype.writeEarlyHints = function writeEarlyHints(hints, cb) {
 
   validateObject(hints, 'hints');
 
-  if (hints.link === null || hints.link === undefined) {
-    return;
-  }
-
-  const link = validateLinkHeaderValue(hints.link);
-
-  if (link.length === 0) {
-    return;
-  }
-
-  head += 'Link: ' + link + '\r\n';
-
   for (const key of ObjectKeys(hints)) {
-    if (key !== 'link') {
+    const hint = hints[key];
+    validateHeaderName(key);
+    validateHeaderValue(key, hint);
+
+    if (ArrayIsArray(hint)) {
+      for (let i = 0; i < hint.length; i++) {
+        head += key + ': ' + hint[i] + '\r\n';
+      }
```

Comment:
Nit: Using `array.prototype.reduce` might make this line much more readable.

Reply:
I think this is more readable.